vng_api_common/filtersets.py

```diff
@@ -25,3 +25,13 @@ class FilterSet(filterset.FilterSet):
     """
 
     FILTER_DEFAULTS = FILTER_FOR_DBFIELD_DEFAULTS
+
+    @classmethod
+    def filter_for_field(cls, field, field_name, lookup_expr=None):
+        """
+        Add help texts for model field filters
+        """
+        filter = super().filter_for_field(field, field_name, lookup_expr)
```

Comment:
Will not let this be blocking for this PR (because it's not much of an issue in this case), but generally I'd avoid naming variables after builtins, because this can cause confusion if you try to use that builtin in the same part of the code

```python
>>> filter
<class 'filter'>
```